src/analytics/private/sqls/redshift/event-v2.sql

```diff
@@ -89,12 +95,12 @@ CREATE TABLE IF NOT EXISTS {{schema}}.event_v2 (
     scroll_engagement_time_msec bigint,
```

Comment:
Can we add a field defined as COALESCE(user_id, user_pseudo_id)? The downstream reporting operations need such a field, but if it is defined in the MV (materialized view), it will prevent the MV from being incrementally updated.